### PR TITLE
Global readme

### LINKED ISSUE #83: Update Global README.md
**Todo**
- [ ] Research a few readme formats
- [ ] Update the global readme to be descriptive whilst being easy to understand
- [ ] Look at the nested directories and link their readme's in from the global readme
- [ ] Add the ERD, Architecture Diagram to Readme
- [ ] Add badges to the Readme.

••Story••
Making the readme better and more descriptive will make it easier for anyone to navigate the project page

### PR DESCRIPTION
**Pull request by danfh00**
Adds details of the project to the global readme
Closes #83

### CODE REVIEW

File: Diagrams/Hermes_Logo.png
Comment (A-Faris) [inline Diagrams/Hermes_Logo.png:1]: Delete this

File: README.md [hunk 1]
```diff
@@ -1 +1,140 @@
-# sales-tracker-hermes
\ No newline at end of file
+# <img src='./Diagrams/Hermes_Logo.png' width='600'>
```
Comment (A-Faris) [inline README.md]:
```suggestion
# <img src='./Dashboard/logo/hermes_logo.png' width='600'>
```

Comment (A-Faris) [reply README.md]: Put the badges first

File: Diagrams/Hermes_Logo.png
Comment (danfh00) [reply Diagrams/Hermes_Logo.png:1]: Updated logo

File: README.md (hunk 1 above)
Comment (danfh00) [reply README.md]: done

**Review by A-Faris — APPROVED** [review]
README includes all the key information

**Pull request merged**